autolink/postlink/appDelegateLinker.js

```diff
@@ -153,7 +153,11 @@ class AppDelegateLinker {
       /rootViewController\.view\s+=\s+rootView;\s+/,
       /self.window.rootViewController\s+=\s+rootViewController;\s+/,
       /\[self.window\s+makeKeyAndVisible];\s+/,
+      // Added from RN 0.69
+      /NSDictionary\s+\*initProps\s+=\s+\[self prepareInitialProps];\s+/,
+      /UIView \*rootView = RCTAppSetupDefaultRootView\(bridge, @".*", initProps\);/,
```

Comment:
I don't think we need this as we construct the views within RNN

Reply:
Yes, with `Added from RN 0.69` comment I meant these are _new lines to be removed_.